.vscode/settings.json

```diff
@@ -0,0 +1,20 @@
+{
+  // 1. Set Tailwind CSS as the language mode for CSS files
+  "files.associations": {
+    "*.css": "tailwindcss"
+  },
+
+  // 2. Configure Prettier as the default formatter for all relevant files
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+
+  // 3. Enable ESLint to auto-fix issues on save
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": "explicit"
+  },
+
+  // 4. Silence the default VS Code CSS validator
+  "css.validate": false,
+  "less.validate": false,
+  "scss.validate": false
+}
```

Comment:
Nice, this will be helpful! Maybe let's also add something like the below to the .gitignore so that if anyone has their own personal vs code settings it isn't accidentally committed:
```
.vscode/*
!.vscode/settings.json
```

Reply:
https://github.com/azavea/echo-locator/pull/667/commits/0549ba9eb5a0a9e87b02c0ba0467b626d39fff85